modules/client/templates/scripts/forseti-client/run_forseti.sh.tpl

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
Does the client need this script?

Reply:
I don't think client needs this script. Removed.